frontend/src/metabase/services.js

```diff
@@ -35,6 +35,8 @@ export const StoreApi = {
 export function maybeUsePivotEndpoint(api, card, metadata) {
   const question = new Question(card, metadata);
 
+  // we need to pass pivot_rows & pivot_cols only for ad-hoc queries endpoints
+  // in other cases the BE extracts these options from the viz settings
   function wrap(api) {
```

Comment:
This is somewhat costly operation, so let's not do it for endpoints which don't use these params